modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -65,22 +41,23 @@ class Configuration(object):
     sign the HTTP requests with the RSA-SSA-PSS signature algorithm, and set the expiration time
     of the signature to 5 minutes after the signature has been created.
       conf = {{{packageName}}}.Configuration(
-        key_id='my-key-id',
-        private_key_path='rsa.pem',
-        signing_scheme=signing.SCHEME_HS2019,
-        signing_algorithm=signing.ALGORITHM_RSASSA_PSS,
-        signed_headers=[signing.HEADER_REQUEST_TARGET, signing.HEADER_CREATED,
-                        signing.HEADER_EXPIRES, signing.HEADER_HOST, signing.HEADER_DATE,
-                        signing.HEADER_DIGEST, 'Content-Type']
-        signature_max_validity=timedelta(minutes=5),
+        signing_info = {{{packageName}}}.HttpSigningConfiguration(
```

Comment:
Can you update this to `{{{packageName}}}.signing.HttpSigningConfiguration`?

Reply:
done